data/categories.txt

```diff
@@ -6594,7 +6772,7 @@ sv:Vindskydd
 th:ที่อยู่อาศัย
 tr:Barınak
 uk:Бівуачний навіс
-vi:Chổ nương thân
+vi:Chổ nương thân
```

Comment:
Typo?

Reply:
Unicode symbols were properly merged in the editor here )